Src/AutoFakeItEasy.FakeItEasy4UnitTest/AutoFakeItEasy.FakeItEasy4UnitTest.csproj

```diff
@@ -19,7 +19,7 @@
   <ItemGroup>
     <PackageReference Include="FakeItEasy" Version="[4.0.0]" Condition=" '$(TargetFramework)'=='net452' " />
     <PackageReference Include="FakeItEasy" Version="[4.0.0]" Condition=" '$(TargetFramework)'=='netcoreapp1.1' " />
-    <PackageReference Include="FakeItEasy" Version="[4.9.0]" Condition=" '$(TargetFramework)'=='netcoreapp2.0' " />
+    <PackageReference Include="FakeItEasy" Version="[4.9.0]" Condition=" '$(TargetFramework)'=='netcoreapp2.1' " />
```

Comment:
Since .NET Core 2.1 is out of support, wouldn't be better to use `netcoreapp3.1`?

Reply:
I think it's more a question which targets are supported by `FakeItEasy 4.9.0`. But you're right we should move the supported .NET Core versions up. Currently the .NET Core 1 is causing a lot of issues.